apps/www/styles/globals.css

```
@@ -231,6 +243,10 @@
   }
 }
 
+@utility cursor-pointer {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)